packages/cloud-cognitive/src/components/CreateModal/CreateModal.stories.js

```diff
@@ -0,0 +1,271 @@
+        id="1"
+        key="form-field-1"
+        labelText="Text input label"
+        helperText="Helper text goes here"
+        placeholder="Placeholder"
+      />
+      <NumberInput
+        id="2"
+        className="create-modal--storybook-input"
+        label="Number input label"
+        helperText="Optional helper text goes here"
+        min={0}
+        max={50}
+        value={1}
+      />
+      <RadioButtonGroup
+        legendText="Radio button legend text goes here"
+        name="radio-button-group"
+        defaultSelected="radio-1">
+        <RadioButton labelText="Radio-1" value="radio-1" id="radio-1" />
+        <RadioButton labelText="Radio-2" value="radio-2" id="radio-2" />
+        <RadioButton labelText="Radio-3" value="radio-3" id="radio-3" />
+      </RadioButtonGroup>
+      <DatePicker datePickerType="single">
+        <DatePickerInput
+          placeholder="mm/dd/yyyy"
+          labelText="Date picker input label"
+          id="date-picker-single"
+        />
+      </DatePicker>
+    </>
+  ),
+};
+
+export const DefaultWithDarkTheme = Template.bind({});
+DefaultWithDarkTheme.storyName = 'Create Modal using dark theme';
+DefaultWithDarkTheme.args = {
+  story: DefaultWithDarkTheme,
+  className: 'sb--use-carbon-theme-g90',
+  ...defaultProps,
+  children: (
+    <>
+      <TextInput
+        id="1"
+        key="form-field-1"
+        labelText="Text input label"
+        helperText="Helper text goes here"
+        placeholder="Placeholder"
+      />
+      <Dropdown
+        id="default"
+        titleText="Dropdown label"
+        helperText="This is some helper text"
+        label="Dropdown menu options"
+        items={['Option 0', 'Option 1', 'Option 2']}
+      />
+      <TextArea
+        id="2"
+        placeholder="Placeholder text"
+        labelText="Text area label"
+        helperText="Optional helper text"
+      />
+    </>
+  ),
+};
+
+export const WithStateManager = TemplateWithStateManager.bind({});
+WithStateManager.storyName = 'Create Modal with state manager';
+WithStateManager.args = {
+  story: WithStateManager,
+  ...defaultProps,
+  children: (
+    <>
+      <TextInput
+        id="1"
+        key="form-field-1"
+        labelText="Text input label"
+        helperText="Helper text goes here"
+        placeholder="Placeholder"
+      />
+      <NumberInput
+        id="2"
+        className="create-modal--storybook-input"
+        label="Number input label"
+        helperText="Optional helper text goes here"
+        min={0}
+        max={50}
+        value={1}
+      />
+      <Dropdown
+        id="default"
+        titleText="Dropdown label"
+        helperText="This is some helper text"
+        label="Dropdown menu options"
+        items={['Option 0', 'Option 1', 'Option 2']}
+      />
+      <TextArea
+        id="3"
+        placeholder="Placeholder text"
+        labelText="Text area label"
+        helperText="Optional helper text"
+      />
+    </>
+  ),
+};
```

Comment:
I feel like the ways the stories are created here are very verbose and not how the docs mention creating them. doing it this way also made all the controls be a json output when everything should be controlled through the storybooks template API and not our own. 

https://storybook.js.org/docs/react/writing-stories/args